clients/client-python/gravitino/catalog/base_schema_catalog.py

```diff
@@ -53,43 +54,42 @@ def __init__(
             _properties=properties,
             _audit=audit,
         )
+        self.namespace = namespace
+        self.schema_namespace = Namespace.of(namespace.level(0), name)
         self.rest_client = rest_client
 
     def as_schemas(self):
         return self
 
-    def list_schemas(self, namespace: Namespace) -> [NameIdentifier]:
+    def list_schemas(self) -> List[str]:
         """List all the schemas under the given catalog namespace.
 
-        Args:
-             namespace: The namespace of the catalog.
-
         Raises:
             NoSuchCatalogException if the catalog with specified namespace does not exist.
 
         Returns:
-             A list of {@link NameIdentifier} of the schemas under the given catalog namespace.
+             A list of schema names (as strings) under the given catalog namespace.
         """
-        Namespace.check_schema(namespace)
+        Namespace.check_schema(self.schema_namespace)
         resp = self.rest_client.get(
-            BaseSchemaCatalog.format_schema_request_path(namespace)
+            BaseSchemaCatalog.format_schema_request_path(self.schema_namespace)
         )
         entity_list_response = EntityListResponse.from_json(
             resp.body, infer_missing=True
         )
         entity_list_response.validate()
-        return entity_list_response.identifiers()
+        return [str(identifier) for identifier in entity_list_response.identifiers()]
```

Comment:
I'm not sure whether the return value is an array of schema name, or an array of identifier's toString() value; Please double check

Reply:
The Python method list_schemas now returns an array of schema names. The change [str(identifier) for identifier in entity_list_response.identifiers()] ensures that it matches the Java method listSchemas, which also returns an array of schema names by invoking NameIdentifier::name.